modin/experimental/engines/omnisci_on_native/test/utils.py

```diff
@@ -66,6 +71,40 @@ def omnisci_comparator(df1, df2):
     )
 
 
+def align_datetime_dtypes(*dfs):
+    """
+    Make all of the passed frames have DateTime dtype for the same columns.
+
+    Cast column type of the certain frame to the DateTime type if any frame in
+    the `dfs` sequence has DateTime type for this column.
+
+    Parameters
+    ----------
+    *dfs : iterable of DataFrames
+        DataFrames to align DateTime dtypes.
+
+    Notes
+    -----
+    Passed Modin frames may be casted to pandas in the result.
+    """
+    datetime_cols = {}
+    for df_dtypes in zip(*tuple(df.dtypes.items() for df in dfs)):
+        for col, dtype in df_dtypes:
+            if is_datetime64_any_dtype(dtype):
+                datetime_cols[col] = dtype
+                break
```

Comment:
I don't get that `break`, what if any other pair of `col/dtype` is a datetime64? why do you stop at the first _column_ instead of checking all of them?